OpenEdXMobile/src/test/java/org/edx/mobile/test/ConfigTests.java

```diff
@@ -5,16 +5,18 @@
 import com.google.gson.JsonPrimitive;
 
 import org.edx.mobile.util.Config;
-
+import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
-import org.junit.Test;
 
-import java.util.Arrays;
 import java.util.ArrayList;
+import java.util.Arrays;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
Lets keep it as is, cuz, whenever an auto-import is applied this'll happen to the wilcard imports.